content/source/docs/enterprise/api/run.html.md

```diff
@@ -41,11 +43,13 @@ Status  | Response                               | Reason
 [200][] | [JSON API document][] (`type: "runs"`) | Successfully created a run
 [404][] | [JSON API error object][]              | Organization or workspace not found, or user unauthorized to perform action
 [422][] | [JSON API error object][]              | Validation errors
+[401][] | [JSON API error object][]              | Tried to create run with organization token (use a user or team token instead)
```

Comment:
Let's do these in order, it makes it easier when scanning.

Reply:
I went back and forth on that! This is sorrrt of listed in order of the frequency you'll get that response, but yeah, maybe that's overthinking it and numerical order is more predictable.